site/en/run/build.md

```diff
@@ -433,6 +433,20 @@ BCR through internet. Add the following line to your `.bazelrc`:
 ```python
 common --registry="path/to/local/bcr/registry"
 ```
+Note that in case you don't have an 'source' server with Internet connectivity to execute the above procedure, the --dist_dir option still works in Bazel 7. This might be a path to follow in case if you don't have access. The --repository_cache option is preferred in case you don't experience this constraint. Some considerations :
+- The packages will need to be downloaded manually using a web browser. This sounds more dramatic as it sounds as only a dozen or so packages are to be downloaded. The following entries could be added to the .bazelrc file or the ~/.bazelrc can be used instead on each of the target platforms without the _:windows_ and _:linux_ extensions on the tags :
+```
+build:windows --distdir=C:\\path\\to\\repository
+test:linux --distdir=/path/to/repository
```

Comment:
What does the `/path/to/repository` mean?

Reply:
/path/to/repository is the path where you hold the downloaded repos, e.g. build:windows --distdir=c:\Users\johndoe\bazel_repos

Reply:
If Bzlmod is enabled, Bazel will likely try to access BCR anyway, so this approach probably won't work.

Reply:
Yes indeed, you are absolutely right. Apologies for not specifying this. I have downloaded the BCR and added to the .bazelrc
- windows : common --registry=file:///Users.../source/bazel-central-registry-main
- linux        : common --registry=file:////....../bazel-central-registry-main